readme.md

```diff
@@ -36,9 +41,13 @@ You can now access the server at [http://0.0.0.0:4000](http://0.0.0.0:4000)
 ### Locally
 
 You'll need:
+
 - Python 3.8 or later
 - Ruby 2.7.7
 
+Optionally, to run the linters, you'll need `yarn` (easy way is via
```

Comment:
I'd say we take out optional  and say its required so we can have everyone running these checks